multipledispatch/tests/test_conflict.py

```diff
@@ -60,3 +61,46 @@ def test_ordering():
 
 def test_type_mro():
     assert super_signature([[object], [type]]) == [type]
+
+
+def test_supercedes_variadic():
+    assert supercedes((Variadic[B],), (Variadic[A],))
+    assert supercedes((B, Variadic[A]), (Variadic[A],))
+    assert supercedes((Variadic[A],), (Variadic[(A, C)],))
+    assert supercedes((A, B, Variadic[C]), (Variadic[object],))
+    assert supercedes((A, Variadic[B]), (Variadic[A],))
+    assert supercedes(tuple([]), (Variadic[A],))
```

Comment:
I actually really appreciate not using a tuple literal here. It makes this easier to read, thanks.